pkgs/by-name/cy/cy/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+  go,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cy";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "cfoust";
+    repo = "cy";
+    rev = "v1.0.0";
+    hash = "sha256-7tqvJlm97upgj/ynxKtoPpe4hzj7OwRILLIOpcZ42hE";
+  };
+
+  buildInputs = [ go ];
+
+  buildPhase = ''
+    export GOCACHE=$PWD/.cache/go-build
+    export GOPATH=$PWD
+    go install ./cmd/cy/
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -v $GOPATH/bin/cy $out/bin/
```

Comment:
prefer `install -Dm755 -t $out/bin $GOPATH/bin/cy`

Reply:
included these changes, thanks for the feedback